app/page.tsx

```diff
@@ -88,6 +119,7 @@ export default function HomePage() {
         orderDirection: OrderDirection.Desc,
         title_contains_nocase: debouncedSearch,
         creator_in: AI_AGENTS_ALLOWLIST,
+        category: category,
```

Comment:
We could reduce verbosity here

```suggestion
        category,
```

Reply:
true